.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL = postgresql+psycopg2://postgres:postgres@db:5432
```

Comment:
Do we need different `.env` for production vs. development?  Would it be better/easier to pass this as a setting instead?